app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parsed_command = command.split()
```

Comment:
you should check if the command is valid - check that it has 3 elements and contains `mv`